### PR TITLE
Add supply migration for kava-8

### PR DESCRIPTION
**Pull request by karzak**
Adds SWP token to `supply.GenesisState` so that `supply` module invariants pass

### CODE REVIEW

File: migrate/v0_15/migrate.go
Comment (nddeluca) [inline migrate/v0_15/migrate.go:269]: Comment needs to be updated?

File: migrate/v0_15/migrate_test.go
Comment (nddeluca) [inline migrate/v0_15/migrate_test.go:39]: Nice like this test 👍

File: migrate/v0_15/migrate.go
Comment (nddeluca) [inline migrate/v0_15/migrate.go:82]: 250 million looks correct. @rhuairahrighairidh agree?

Comment (nddeluca) [reply migrate/v0_15/migrate.go:82]: Might be nice to put in a named constant

**Review by nddeluca — COMMENTED** [review]
Looks good, should wait for second look from @rhuairahrighairidh before merge

File: migrate/v0_15/migrate_test.go
Comment (rhuairahrighairidh) [reply migrate/v0_15/migrate_test.go:39]: seconded

**Review by rhuairahrighairidh — APPROVED** [review]
looks good, tested it locally with a genesis export from a week or so ago

**Pull request merged**
